README.md

```diff
@@ -21,22 +21,12 @@ npx @eslint/create-config
 If you want to use a specific shareable config that is hosted on npm, you can use the `--config` option and specify the package name:
 
 ```bash
-# use `eslint-config-semistandard` shared config
-
-# npm 7+
-npm init @eslint/config -- --config semistandard
-
-# or (`eslint-config` prefix is optional)
-npm init @eslint/config -- --config eslint-config-semistandard
-
-# ⚠️ npm 6.x no extra double-dash:
-npm init @eslint/config --config semistandard
+# use `eslint-config-standard` shared config
+npm init @eslint/config -- --config eslint-config-standard
 ```
 
-The `--config` flag also supports passing in arrays:
+To use an eslintrc-style(deprecated) shared config:
```

Comment:
Maybe  _legacy_ would fit better, since that's the name we use in the main repo to refer to eslintrc features?

```suggestion
To use an eslintrc-style (legacy) shared config:
```